geode-for-redis/README.md

```diff
@@ -34,11 +34,12 @@ Once the locator has started, start at least one server with a command of the fo
 gfsh> start server \
   --name=<serverName> \
   --locators=localhost[<locatorPort>] \
-  --geode-for-redis-port=<geodeForRedisPort> \
-  --geode-for-redis-bind-address=<geodeForRedisBindAddress>
+  --J=-Dgemfire.geode-for-redis-enabled=true \
+  --J=-Dgemfire.geode-for-redis-port=<geodeForRedisPort> \
+  --J=-Dgemfire.geode-for-redis-bind-address=<geodeForRedisBindAddress>
 ```
 
-If any of the options `geode-for-redis-bind-address`, `geode-for-redis-username`, or `geode-for-redis-port` are included, a Geode server with Geode for Redis will be started.
+If the option `geode-for-redis-enabled` is set to `true`, a Geode server with Geode for Redis will be started.
```

Comment:
option once again

Reply:
Yeah, this needs a fix. Must be explicitly enabled now.

Reply:
Except we're getting rid of the Redis README.md now, I think.